src/Controller/PackagistController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the badge-poser package.
+ *
+ * (c) PUGX <http://pugx.github.io/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Controller;
+
+use Packagist\Api\Client as PackagistClient;
+use Packagist\Api\Result\Result as PackagistResult;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class SearchPackagistController.
+ *
+ * @author Giulio De Donato <liuggio@gmail.com>
```

Comment:
I think we can avoid to repeat contributors in every single controller. You should add only your name (or, even better, avoid completely such kind of information. Users can always check for contributors here on github)

Reply:
I copied from the old version and added mine!
If we decide to remove for me there are no problems and i proceed to remove everywhere!